docs/deb_index.md

```diff
@@ -0,0 +1,120 @@
+<!-- Generated with Stardoc: http://skydoc.bazel.build -->
+
+`deb_index` macro
```

Comment:
We can probably drop deb_index from public api docs to not confuse people. `apt.install` is good enough for everyone.

Reply:
Ah! this has to be some leftover, I thought I had changed it to "`apt.install` macro" everywhere. I'll go over it again!

Reply:
LOL the `deb_index.md` file should have been deleted, the macro doc is `apt_macro.md`. Deleting it now...